packages/main/src/ListItemStandard.hbs

```diff
@@ -24,16 +24,10 @@
 {{/inline}}
 
 {{#*inline "imageBegin"}}
-	{{#if hasImageContent}}
-		<div class="ui5-li-imgContent" >
-			<slot name="imageContent"></slot>
+	{{#if hasImage}}
+		<div class="ui5-li-image" >
+			<slot name="image"></slot>
 		</div>
-	{{else}}
-		{{#if displayImage}}
-			<ui5-avatar shape="Square" class="ui5-li-img">
-				<img src="{{image}}" class="ui5-li-img-inner" />
```

Comment:
There is still some CSS for the `ui5-li-img-inner` class in the `ListItem.css` file. Let's move it to the sample to achive the same design of the avatar's image (object-fit: contain). We may also add it to the breaking change notes.